Modules/ITS/src/ITSFhrTask.cxx

```diff
@@ -516,12 +516,33 @@ void ITSFhrTask::monitorData(o2::framework::ProcessingContext& ctx)
       for (auto& digit : digVec[istave][0]) {
         int chip = digit.getChipIndex() % 9;
         mHitPixelID_InStave[istave][0][chip][1000 * digit.getColumn() + digit.getRow()]++;
+        if (mTFCount <= mCutTFForSparse) {
+              int pixelPos[2] = { digit.getColumn() + (1024 * chip) + 1, digit.getRow() + 1 };
+              int BinContent;
+              mStaveHitmap[istave]->GetBinContent(mStaveHitmap[istave]->GetBin(pixelPos),&BinContent); 
+              mStaveHitmap[istave]->SetBinContent(pixelPos,BinContent+1);
```

Comment:
Why don't you use `mStaveHitmap[istave]->Fill(pixelPos)` ? Same for the other places.

Reply:
I thought that ThnSparse does not have Fill() function, but it appeared that It was necessary to fill *Double_t not *int_t. Fixed